client/galaxy/scripts/mvc/tool/tool-genomespace.js

```diff
@@ -0,0 +1,23 @@
+// Provides support for interacting with the GenomeSpace File Browser popup dialogue
+define([], function() {
+
+// tool form templates
+return {
+    openFileBrowser: function( options ) {
+        var GS_UPLOAD_URL = "https://gsui.genomespace.org/jsui/upload/loadUrlToGenomespace.html?getLocation=true";
```

Comment:
@jmchilton Thanks. I've added a galaxy.ini config setting for the GenomeSpace URL now: https://github.com/galaxyproject/galaxy/pull/1814/commits/04a7e7676386b4818796822e24f648ded6c29d67